documentation/docs/pages/operators/introduction.mdx

```diff
@@ -14,13 +14,13 @@ client ───► Gateway ──┘  mix  │  mix  ┌─►mix ───►
                          mix  └─►mix──┘  mix
 ```
 
-If you are new to Nym and want to learn about the Mixnet, explore kickstart options and demos, learn how to integrate with the network, and follow developer tutorials check out the [Developer Portal](https://nymtech.net/docs/developers/).
+If you want to dive deeper into Nym Network architecture, clients and nodes visit the [Network docs](https://nymtech.net/docs/network).
 
-If you want to dive deeper into Nym's architecture, clients, nodes, and SDK examples visit the [technical docs](https://nymtech.net/docs/network).
+If you are want to explore kickstart options and demos, learn how to integrate with the network, and follow developer tutorials check out the [Developer Portal](https://nymtech.net/docs/developers/).
```

Comment:
I'd remove 'and demos', we dont have too many demos yet. Or change it to `examples`?